plugins/uefi/fu-plugin-uefi.c

```diff
@@ -583,6 +585,9 @@ fu_plugin_uefi_ensure_esp_path (FuPlugin *plugin, GError **error)
 		}
 		return TRUE;
 	}
+	require_shim = fu_plugin_get_config_value (plugin, "RequireShim");
+	if (require_shim == NULL || g_strcmp0 (require_shim, "true") == 0)
```

Comment:
It would be good to add something that converts this config value to lower case so it's not magic to know that `True`, `TRUE`, and `true` don't all activate this.

Reply:
I'm hesitant to add `fu_plugin_get_config_boolean` as it's unclear what `NULL` should map to. I've switched this to use `g_ascii_strcasecmp()` instead.